dashboard/app/models/script.rb

```diff
@@ -87,7 +87,8 @@ def starting_level
   # indexed by both id and name. This is cached both in a class
   # variable (ie. in memory in the worker process) and in a
   # distributed cache (Rails.cache)
-  @@script_cache = nil
+  cattr_accessor :script_cache
```

Comment:
maybe i could use some help tracking down why this seemed to work with `script_preload.rb` while the original didn't. `cattr_writer` would probably work the same, i'll try it.